packages/react-search-ui-views/src/__tests__/SearchBox.test.tsx

```diff
@@ -22,7 +33,8 @@ it("renders correctly", () => {
 it("applies 'focused' class when `isFocused` is true", () => {
   const wrapper = shallow(<SearchBox {...requiredProps} isFocused={true} />);
   const downshift = wrapper
-    .dive("Downshift")
```

Comment:
dive does not support arguments like this. Enzyme's types were complaining. The right strategy i believe is using `find` to focus then dive.